src/js/protocols/stm32.js

```diff
@@ -8,19 +8,21 @@
 'use strict';
 
 var STM32_protocol = function () {
-    this.baud;
+    this.baud = null;
     this.options = {};
-    this.callback; // ref
-    this.hex; // ref
-    this.verify_hex;
+    this.callback = null;
+    this.hex = null;
+    this.verify_hex = [];
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'verify_hex' is not in camel case.](https://app.codacy.com/gh/emuflight/EmuConfigurator/pullRequest?prid=5626377)